usaspending_api/common/mixins.py

```diff
@@ -254,6 +256,70 @@ class SuperLoggingMixin(LoggingMixin):
 
     events_logger = logging.getLogger("events")
 
+    """Mixin to log requests - customized to disable DB logging, remove this method to re-enable"""
```

Comment:
So this is basically duplicating some library code, but instead of sending it to the DB it's solely keeping it around for logging to `events.log`; Basically we're not really _disabling_ anything per se, we're writing some functionality to log this data to a file and omit logging it to the database.